MAUI/ImageEditor/toolbar-customization.md

```diff
@@ -247,12 +344,12 @@ The ImageEditor automatically generates the toolbars. To add the toolbars manual
                 <imageEditor:ImageEditorToolbarGroupItem Alignment="Start">
```

Comment:
Add note:

When you create a new toolbar item, you can use pre-existing built-in names for the item, which will automatically generate the icon view. If you want to introduce new icons or modify the current one, you can use the `View` property of the `ImageEditorToolbarItem` class to define the item view.

Reply:
added.